packages/scaffold/src/partials/w3m-account-nfts-widget/index.ts

```diff
@@ -36,12 +36,12 @@ export class W3mAccountNftsWidget extends LitElement {
         justifyContent="center"
         flexDirection="column"
       >
-        <wui-text variant="paragraph-500" align="center" color="fg-100">No NFTs yet</wui-text>
+        <wui-text variant="paragraph-500" align="center" color="fg-100">Coming soon</wui-text>
         <wui-text variant="small-400" align="center" color="fg-200"
-          >Transfer from another wallets to get started</wui-text
+          >Stay tuned for our upcoming NFT feature</wui-text
         >
       </wui-flex>
-      <wui-link @click=${this.onReceiveClick.bind(this)}>Receive NFTs</wui-link>
+      <wui-link @click=${this.onReceiveClick.bind(this)}>Receive funds</wui-link>
```

Comment:
Are the copies from the designs? @svenvoskamp

Reply:
This is actually placeholder copy, because we currently don't support NFTs yet